app/services/person_constructor.rb

```diff
@@ -58,24 +63,21 @@ def flatten_remote_fields
   end
 
   def strip_whitespace_from_values
-    params.merge!(params){ |k, v1| v1.try(:strip) || v1 }
+    params.merge!(params){ |k, v| v.try(:strip) || v }
```

Comment:
Sure, but the idiomatic way to show discarded arguments in a block is usually to use underscores.  I'll update.

`params.merge!(params){ |k, v, _| v.try(:strip) || v }`